### PR TITLE
update the alert email copy

### PR DESCRIPTION
**Pull request by pnavarrc**
[Trello](https://trello.com/c/Cm0vZo44/549-alert-emails-add-copy-to-indicate-the-frequency-of-the-emails) - Add copy to indicate the frequency of email alerts

This PR adds a note telling users that we only send email alerts on Mondays and Thursdays (users email us asking why sometimes it doesn't work). I sent myself a test email, this is what it looks like:

image

### CODE REVIEW

File: scripts/alert_emails/template.html [hunk 1]
```diff
@@ -20,162 +20,271 @@
...
+                                        <div
+                                            style="line-height:inherit;vertical-align:middle;border-width:0!important;border-style:none!important;border-color:white!important">
+                                            <p
+                                                style="Margin-top:0;Margin-bottom:0;font-style:normal;font-weight:normal;font-family:Arial,Helvetica,sans-serif;text-align:center;font-size:15px;line-height:21px">
+                                                <span style="line-height:inherit">
+                                                    <span style="line-height:inherit;color:#4f4f4f">To avoid
+                                                        overwhelming your inbox, we only send out Alerts on Mondays and
+                                                        Thursdays. For the most up-to-date risk
+                                                        assessments, check out the <a
+                                                            href="https://covidactnow.org/about"
```
Comment (mikelehen) [inline scripts/alert_emails/template.html:207]: Why are we linking to the about page?

**Review by mikelehen — APPROVED** [review]
Looks good except I'm not sure why it's linking to the about page?

File: scripts/alert_emails/template.html (hunk 1 above)
Comment (pnavarrc) [reply scripts/alert_emails/template.html]: Fixed!

**Comment by pnavarrc** [comment]
[Alert emails: Add copy to indicate the frequency of the emails](https://trello.com/c/Cm0vZo44/549-alert-emails-add-copy-to-indicate-the-frequency-of-the-emails)

**Pull request merged**